### ISSUE TITLE
Tailwind css class inside components are not inlined server side

### ISSUE THREAD

**Issue by manzonif** [angular/universal#2421]
# 🐞 Bug report

#### What modules are related to this issue?

- [ ] aspnetcore-engine
- [ ] builders
- [ ] common
- [x ] express-engine
- [ ] hapi-engine

### Is this a regression?

 Yes, the previous version in which this bug was not present was: ....

### Description
I'm not sure if this is a bug or normal behavior or a possible feature.

My instance of ngExpressEngine has inlineCriticalCss set to true. 
Inspecting the source code of the page, I realized that angular universal correctly inline css classes of each component included in the respective scss files, but not the tailwind css classes, (I only find those in the style.css file).
I would have expected a <style> in the head of the page that would include all the tailwind classes used, without repetitions.

## 🔥 Exception or Error

<pre><code>

</code></pre>

## 🌍 Your Environment

<pre><code>

</code></pre>

**Comment by manzonif** [comment]
Hi @alan-agius4, I'll try. 
It would be useful to have some sort of starter repo to clone. Maybe in the future.

**Comment by alan-agius4** [comment]
You can run the below to get a starter repo

```
ng new repro-app
ng add @nguniversal/express-engine
```

**Comment by manzonif** [comment]
> You can run the below to get a starter repo
> 
> ```
> ng new repro-app
> ng add @nguniversal/express-engine
> ```

Cool!

With the repro-app I was unable to reproduce the issue.
With my project, however, I am stuck with angular version 12.0.1, due to [this](https://github.com/angular/angular/issues/44004). So it could be that with 12.2.12 the problem has gone away.
However, I would wait before closing the matter.

Another question now.
Looking inside the html output of the repro-app, I found some duplicate css. 
Basically, the part of css that is inserted in the index.htm, is also reported in the <style> created during the rendering on the server side. is this a required behavior?

![tailwind-1](https://user-images.githubusercontent.com/8948699/139791742-93cfa3a5-632d-41b6-9060-0b4d1f6312b9.PNG)

**Comment by manzonif** [comment]
solved.
